assets/app/features/upgrade/upgrade.controller.js

```diff
@@ -18,7 +18,8 @@
         vm.steps = {
             list: [],
             nextStep: nextStep,
-            isLastStep: isLastStep
+            isLastStep: isLastStep,
+            stepFinished: upgradeStepsFactory.stepFinished
```

Comment:
THANK YOU!

I knew it was missing somewhere and could not find it for the life of me.